config/branding.yml.sample

```diff
@@ -32,7 +32,7 @@ defaults: &defaults
     #          Comment this line out if you want to use DMPRoadmap's built-in contact form.
     contact_us_url: 'https://somewhere.com/contact-us'
 
-  application:
+  application: &application_defaults
```

Comment:
Added these variable names so you can do things like:
```yaml
development:
  <<: *defaults
  application:
    <<: *application_defaults
    name: 'DMPTool-Dev'
```
Without the `<<: *application_defaults` line, a check of `Rails.configuration.branding[:application].inspect` shows that it ONLY contains the `name:` param

Reply:
Oh cool!
That's much better than the way I was managing those differences, and it's nice to have the template in there for everybody.
Thanks @briri